resources/core/charts/cluster-users/templates/rbac-roles.yaml

```diff
@@ -124,3 +125,78 @@ subjects:
   name: {{ . }}
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+
+# kyma-edit = edit permissions on kyma resources
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyma-edit
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-edit: "true"
+  annotations:
+    "helm.sh/hook-weight": "0"
+rules:
+- apiGroups: ["authentication.kyma-project.io", "gateway.kyma-project.io", "kubeless.io", "applicationconnector.kyma-project.io", "servicecatalog.kyma-project.io", "kyma.cx", "authentication.istio.io", "config.istio.io", "ui.kyma.cx", "ui.kyma-project.io", "networking.istio.io"]
+  resources: ["*"]
+  verbs: ["create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"]
+- apiGroups: ["authorization.k8s.io"]
+  resources: ["selfsubjectrulesreviews", "selfsubjectaccessreviews"]
+  verbs: ["create"]
+- nonResourceURLs: ["*"] #give access to all non resource urls
+  verbs: ["*"]
+
+# Cluster developer role
+# Kyma-developer = kyma-edit + k8s-edit
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kyma-developer
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+  annotations:
+    "helm.sh/hook-weight": "0"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.authorization.k8s.io/aggregate-to-edit: "true"
+  - matchLabels:
+      rbac.authorization.kyma-project.io/aggregate-to-kyma-edit: "true"
+rules: []
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Kyma developer role should be granted only in namespaces. The developer should not be able to create deployment in kyma-system. Please do not create this binding. In the test, you should grant this role in one namespace and test access in that namespace (you can use default).

Reply:
Moved Binding into test, added namespace as an env to the test